pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterMemBasedBrokerQueryKillingTest.java

```diff
@@ -207,8 +207,6 @@ protected TableConfig createOfflineTableConfig() {
         .build();
   }
 
-
-  @Test
```

Comment:
- The Linter fails.
- How about adding a TODO referencing https://github.com/apache/pinot/issues/11099 and saying re-enable the test after investigation?

Reply:
fixed and added, thanks